README.md

```diff
@@ -137,6 +138,18 @@ A live ebuild is also available in the [overlay](https://github.com/stefantalpal
 Pony requires LLVM 3.6, 3.7 or 3.8. Please note that **LLVM 3.7.0 does not work**. If you are using LLVM 3.7.x, you need to use 3.7.1. If you are using LLVM 3.6.x, make sure to use 3.6.2.
 
 ## Building on Linux
+
+Install and setup Git (if not already done get an Account here: https://github.com/ )
```

Comment:
You shouldn't need to have a GitHub account to clone a public git repo from GitHub - you can clone anonymously as long as the repo is public.  Because of this, I think the instructions about creating an account and setting up your `git config` are kind of out of scope for cloning and building `ponyc`.